scripts/cybershake/auto_submit.py

```diff
@@ -61,7 +62,7 @@ def submit_task(
     models=None,
 ):
     task_logger = workflow_logger.get_task_logger(parent_logger, run_name, proc_type)
-
+    verification_dir = os.path.join(sim_dir, 'Verification')
```

Comment:
Should be moved to qcore.simulation_structure, especially if other functions/scripts are going to use the Verification folder